runtime/sema/member_accesses.go

```diff
@@ -57,15 +57,22 @@ func (m *MemberAccesses) Find(pos Position) *MemberAccess {
 	if interval == nil {
 		return nil
 	}
-	access := value.(MemberAccess)
+	access, ok := value.(MemberAccess)
+	if !ok {
+		return nil
+	}
 	return &access
 }
 
 func (m *MemberAccesses) All() []MemberAccess {
 	values := m.tree.Values()
 	accesses := make([]MemberAccess, len(values))
 	for i, value := range values {
-		accesses[i] = value.(MemberAccess)
+		access, ok := value.(MemberAccess)
+		if !ok {
+			return nil
```

Comment:
would `continue` be a better option? I'm ok with both.

Reply:
I think this depends on what we'd like this function to do. If one of the elements of `values` is not a `MemberAccess` even though this function is meant to be called on a `MemberAccesses` collection, that feels like a significant enough issue that we'd like to fail the entire operation and return `nil`. However, if you feel like we should just recover as much as possible and return those values that are valid then we can `continue`.

Reply:
I felt returning `nil` would be somewhat closer to returning an empty array (the result of this is more likely to be looped over the `range`). Maybe if we want to completely fail the operation then panicking with a dedicated error would be even better I guess.
My initial thought was, if we are silently failing, we could make it to be resilient as much as possible.